sdk/trace/trace_test.go

```diff
@@ -2005,52 +2009,104 @@ func TestAddLinkWithInvalidSpanContext(t *testing.T) {
 		parent:               sc.WithRemote(true),
 		links:                nil,
 		spanKind:             trace.SpanKindInternal,
-		instrumentationScope: instrumentation.Scope{Name: "AddSpanWithInvalidSpanContext"},
-	}
-	got, err := endSpan(te, span)
-	if err != nil {
-		t.Fatal(err)
+		instrumentationScope: instrumentation.Scope{Name: "AddLinkToNonRecordingSpan"},
 	}
+
 	if diff := cmpDiff(got, want); diff != "" {
-		t.Errorf("AddLinkWithInvalidSpanContext: -got +want %s", diff)
+		t.Errorf("AddLinkToNonRecordingSpan: -got +want %s", diff)
 	}
 }
 
 func TestAddLink(t *testing.T) {
```

Comment:
Also refactored links tests to table-tests, because of too much copy paste with single line changed